GUI/DeusEx/Classes/MenuChoice_RefuseMelee.uc

```diff
@@ -0,0 +1,17 @@
+class MenuChoice_RefuseMelee extends MenuChoice_ItemRefusal;
+
+defaultproperties
+{
+    enabled=False
+    defaultvalue=False
```

Comment:
I actually think melee should be enabled by default?

Reply:
I think item refusal should be an opt-in feature. Better to not make assumptions about what the player wants.

Reply:
Well current defaults mean carcasses don't have melee weapons anyways, so I think this one is safe. Also cigarettes and zyme would probably be good to enable by default.

Reply:
I would expect turning off enemies throwing their weapons to make the game behave normally, unless I chose to set them to refused.

Reply:
I guess we'll wait to change both defaults at the same time